c7n/cli.py

```diff
@@ -73,9 +73,13 @@ def _default_options(p, blacklist=""):
     config.add_argument("-t", "--resource", default=None, dest='resource_type',
                         help="Only use policies with the given resource type")
 
-    p.add_argument("-v", "--verbose", action="store_true",
-                   help="Verbose logging")
-    p.add_argument("--debug", default=False, help=argparse.SUPPRESS,
+    config.add_argument("-v", "--verbose", action="count", help="Verbose logging")
+    if 'quiet' not in blacklist:
```

Comment:
That approach means I can't keep the quiet option next to the verbose option in the help text because we are using argument groups.  I think they should be next to each other.  Agree?  (Speaking of that, `--verbose`, `--quiet`, and `--debug` should exist in their own option group.  I'll add that now)

The blacklists do feel clunky.  I wonder if we should switch to whitelists at this point, and try to move any option that is common to more than one subcommand into the `_default_options` function.